src/common/arrow/Cargo.toml

```diff
@@ -34,7 +34,7 @@ simd = ["arrow/simd"]
 # Workspace dependencies
 
 # Crates.io dependencies
-arrow = { package = "arrow2", version = "0.14.0", default-features = false, features = [
+arrow = { package = "arrow2", git = "https://github.com/RinChanNOWWW/arrow2", rev = "8bd6417", default-features = false, features = [
```

Comment:
better to use upstream source.

Reply:
Changed. And we may need to change to version or other rev when the PR is merged in arrow2.

Reply:
It seems unable to get the dependency if the rev not belongs to the repo.

Reply:
can we merge `8bd6417` to arrow2?

Reply:
It's still in progress.  https://github.com/jorgecarleitao/arrow2/pull/1287